README.md

```diff
@@ -145,3 +145,15 @@ To stop the docker container open another terminal, run `docker ps`, get the ID
 and replace the docker container ID where the `<ID>` placeholder is.  I hope to find a more graceful way to stop the application 
 in the future.
    
+## Local Development
```

Comment:
Added some local development instructions...because I forgot them when doing this work :laughing: .